README.md

```diff
@@ -60,41 +60,53 @@ DTOs are described at [api.ts](./api.ts).
 >
 > All paths are prefixed with `/api/v1`
 
-- `GET /` - web server health check. Returns 200 OK.
+- **`GET`** `/`
```

Comment:
@outoftardis Should be able to help us with this.

Reply:
I don't think it's a good idea to put all the API info in readme, this is not a good practice. It's better to have it in a separate file and refer to it from readme.

There are also a few things I'd change in how this info is presented. You can merge this PR, and I'll work on changing the way this info is presented in a separate PR

Reply:
@appetrosyan @0x009922 see https://github.com/soramitsu/iroha2-block-explorer-backend/pull/26